sygnal/__init__.py

```diff
@@ -287,6 +314,17 @@ def setup():
     else:
         logging.basicConfig()
 
+    if cfg.has_option("metrics", "sentry_dsn"):
```

Comment:
is there not some "default config" or "sample config" that this needs adding to?

Reply:
I've added it to sample config.